File: livetester-service/src/main/java/com/yorosis/livetester/grid/vo/PaginationVO.java
package com.yorosis.livetester.grid.vo;



import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PaginationVO {
	private String gridId;
	private int index;
	private int size;
	private String direction;
	private String columnName;
	private String id;
	private FilterValueVO[] filterValue;
   
}

File: livetester-ui/src/app/yorogrid/pagination-vo.ts
export class PaginationVO {
    gridId: string;
    index: number;
    size: number;
    direction: string;
    columnName: string;
    id: string;
    filterValue: FilterValueVO[] = [];

}

export class FilterValueVO {
    filterIdColumn: string;
    operators: string;
    filterIdColumnValue: string;
}

